can/bus.py

```diff
@@ -172,7 +172,7 @@ def send(self, msg: Message, timeout: Optional[float] = None):
 
     def send_periodic(
         self,
-        msgs: Union[Sequence[Message], Message],
+        msgs: Union[Message, Union[List[Message], Tuple[Message, ...]]],
```

Comment:
Any reason for this change? Just curious how it was incorrect, or why this is better?

Reply:
I changed it since that is more in line with what is actually checked and what the error tells the user. But now that I look at it, we should probably do `isinstance(msgs, S)` instead of `isinstance(msgs, (list, tuple))` and keep the old typing.